docs/src/integer.md

```diff
@@ -79,21 +87,37 @@ We give some examples of such functionality.
 
 **Examples**
 
-```@repl
-using AbstractAlgebra # hide
-f = ZZ(12)
-
-h = zero(ZZ)
-k = one(ZZ)
-isone(k) == true
-iszero(f) == false
-U = base_ring(ZZ)
-V = base_ring(f)
-T = parent(f)
-f == deepcopy(f)
-g = f + 12
-h = powmod(f, 12, ZZ(17))
-flag, q = divides(f, ZZ(3))
+```jldoctest
+julia> f = ZZ(12)
+12
+
+julia> h = zero(ZZ)
+0
+
+julia> k = one(ZZ)
+1
+
+julia> isone(k)
+true
+
+julia> iszero(f)
+false
```

Comment:
There are plenty of these `iszero(f)` tests. But I wonder if they are meant to test `iszero(h)` instead?

Actually, if we want to turn this into more of a test (and perhaps less of a manual example...), we could do something like this:
```
julia> map(iszero, [f, h, k])
3-element Array{Bool,1}:
 false
  true
 false
```
or more compactly, this:
```
julia> ( iszero(f), iszero(h), iszero(k) )
(false, true, false)
```

Anyway, I am not particular invested in any of these options; I just wonder why one would use `iszero(f)` over `iszero(h)` or even `iszero(k)`.